_guides/002-install_and_run.md

```diff
@@ -63,6 +63,10 @@ sections:
         title: Dynamic provisioning
         description: Dynamic provisioning
         module: install/kubernetes/dynamic-provisioning
+      -
+        title: Dynamic provisioning (CSI)
+        description: Dynamic provisioning (CSI)
+        module: install/kubernetes/dynamic-provisioning-with-csi
```

Comment:
This hierarchy doesn't feel right. I'd prefer a Kubernetes (CSI) item at the same level as Kubernetes.